test/unit/os/shell_spec.lua

```diff
@@ -112,7 +112,7 @@ describe('shell functions', function()
 
       local argv = ffi.cast('char**',
                         cimported.shell_build_argv(to_cstr('echo &|<>()@^'), nil))
-      eq(ffi.string(argv[0]), '/bin/bash')
+      eq(ffi.string(argv[0]), '/bin/sh')
```

Comment:
I don't think the shell_build_argv tests need to be changed, since the command is not invoked there, it only builds a string. That's why `echo -n` still works there.